package.json

```
@@ -20,7 +20,7 @@
     "emmet-monaco-es": "^4.7.1",
     "escape-html": "1.0.3",
     "hotkeys-js": "3.8.7",
-    "js-base64": "3.6.1",
+    "lz-string": "^1.4.4",
```

@QuevedoIB Thank you for your contribution, in the mean time midudev decides approve or not your PR, we avoid using `^`, I think you may remove it. :+1: 
```suggestion
    "lz-string": "1.4.4",
```